ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/searchUtils.tsx

```diff
@@ -0,0 +1,27 @@
+import qs from 'qs';
+import { SearchFilter } from 'types/search';
+
+export type CveStatusTab = 'Observed' | 'Deferred' | 'False Positive';
+export type EntityTab = 'CVE' | 'Image' | 'Deployment';
+
+export type WorkloadCvesSearch = {
+    cveStatusTab: CveStatusTab;
+    entityTab?: EntityTab;
+    s?: SearchFilter;
+};
+
+function isValidCveStatusTab(cveStatusTab: unknown): cveStatusTab is CveStatusTab {
```

Comment:
TIL: I hadn't seen the `is` TS operator before.

Reply:
Also, something to think about for the future: generalize this check by accepting a list of valid tabs as a param.